### PR TITLE
fixup! README.md: Add a Windows-specific preamble

### PR DESCRIPTION
**Pull request by dscho**
Add another build badge, in addition to the CR/LF one: this one is for
the Windows/macOS/Linux builds.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Git for Windows
 ===============
 
+[![Build status](https://git-for-windows.visualstudio.com/git/_apis/build/status/12?branch=master)](https://git-for-windows.visualstudio.com/git/_build/latest?definitionId=12&branch=master)]
```
Comment (shiftkey) [inline README.md]: The stray `]` at the end of this line is not necessary for rendering the badge, and can be removed:

screen shot 2018-07-06 at 9 59 19 am

Comment (dscho) [reply README.md]: D'oh!

Comment (dscho) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 Git for Windows
 ===============
 
+[![Build status](https://git-for-windows.visualstudio.com/git/_apis/build/status/12?branch=master)](https://git-for-windows.visualstudio.com/git/_build/latest?definitionId=12&branch=master)
```
Comment (shiftkey) [inline README.md]: Can the `Build status` alt text here and on the next line be used to differentiate these builds? It'd be cool if the image itself could embed the build name, but until then some sort of difference will help here without having to click into both...

Comment (dscho) [reply README.md]: Good call. Can you see whether my new proposal addresses your wish?

Comment (nalla) [reply README.md]: You still need to click on both if you are on mobile and can not hover over the buttons, but I guess that’s just a small tadeoff.

**Comment by shiftkey** [comment]
Given this PR change is just documentation, I'm fine with merging but I'll defer to @dscho rather than YOLO it in.

**Comment by dscho** [comment]
@shiftkey for future record: I am fine with you YOLOing it in. When I open a PR that I do not really think is ready to merge just yet, I will make sure to set the title to `[WIP] ...`. In all other cases, I trust the reviewers to merge.

**Pull request merged**